### ISSUE TITLE
Compiling libvips necessary?

### ISSUE THREAD

**Issue by lemmi** [Nico640/docker-unms#76]
I was toying around building the image and the compilation of `libvips` stood out to me. Since alpine ships a current version as well, I was able to build and the image without these lines:
https://github.com/Nico640/docker-unms/blob/b7106820fe11199c422937209d9d31188725105f/Dockerfile#L38-L41

**Comment by Nico640** [comment]
I haven't checked since updating Alpine Linux to 3.15, but the libvips version that shipped with 3.13 was too old for the sharp version used by UISP and since sharp does not offer prebuild libvips binaries for ARMv7 musl, I had to build it from source. So yeah, it's probably not necessary at the moment, however, I like to stay in sync with the versions that the official UISP images uses to minimize these kinds of issues. For example, the official image currently uses libvips 8.12.2, whereas Alpine 3.15 ships with 8.12.1. Maybe I need to rethink this practice at some point or split the image into multiple base images / stages because the fact that building the image can take upwards of an hour is really annoying.

**Comment by lemmi** [comment]
Yes, the build time adds up and this was something that stood out for me. But I can totally understand to play it safe for now, since there is no guarantee this won't run out of sync in the future again.
